app/src/app/allplayerstats/page.tsx

```diff
@@ -26,8 +29,19 @@ export default function AllPlayerStats() {
                 setBookmarks(pids)
             });
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This line should not be required anymore, double check this by removing it and running `npm run lint` to make sure there are no more errors